### ISSUE TITLE
Support for List's default behaviour (swipe across/Multi row delete)

### ISSUE THREAD

**Issue by perbrondum** [rohanrhu/Swipy#1]
WIll you implement, or is it possible to implement, the two enhancements to ForEach OnDelete, like swiping across to perform a default (destructive) action, as well as swipe down to delete multiple rows?  Users are used to these actions on lists,  and we just want to make sure it's possible to implement these as well.

**Comment by rohanrhu** [comment]
Hi, thank you for your feedback.

I just released v1.2.0 ([Purrfectized Button, NavigationLink, ... in Swipy view](https://github.com/rohanrhu/Swipy/commit/791a111381b64e0122edb8c3a9bd9f7ef994cb80)), please update your package. 😇

I didn't understand what you mean; could you explain more? If one of what you mean is swiping harder to perform the rightest-placed action, I'm planning to implement that feature.

**Comment by perbrondum** [comment]
Yeah, that's one our users like to use. it's convenient to swipe harder, avoid the delete button UI (or whatever the default action is) and just get a custom dialog where you can accept the default action.
The other is 'two finger swipe down' that exposes a extra column icon with the selected rows marked, along with a top menu items to perform actions.  This is less important that the first, but really usefull when taking a common action on multiple rows.

**Comment by emog** [comment]
@perbrondum  - Did you manage to make it work for List view with detailview ? I'm trying but when I tap on some button the two buttons are tapped instead of just the right one. If so please share some code.

**Comment by rohanrhu** [comment]
> [@perbrondum](https://github.com/perbrondum) - Did you manage to make it work for List view with detailview ? I'm trying but when I tap on some button the two buttons are tapped instead of just the right one. If so please share some code.

Can you share your code and a video that represents the issue?
